spec/services/data_api/tad_export_spec.rb

```diff
@@ -17,4 +17,13 @@
       expect(result.map { |r| r[:status] }).to match_array(%w[rejected_by_default declined_by_default rejected declined])
     end
   end
+
+  describe '#status' do
+    it 'only includes states that are documented' do
+      documented_columns = DataSetDocumentation.for(described_class)['status']['enum']
+      possible_states_in_export = ApplicationStateChange.states_visible_to_provider.map(&:to_s) + %w[rejected_by_default declined_by_default]
```

Comment:
Should we define a new set on `ApplicationStateChange`?

Reply:
Hmm, not sure - we aren't using it anywhere else I think?

Reply:
Done now!